.github/workflows/update-docs.yml

```diff
@@ -12,15 +12,41 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Pushes Readme file
+      - name: Pushes readme file
         uses: dmnemec/copy_file_to_another_repo_action@main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           API_TOKEN_GITHUB: ${{ secrets.ARC_DOCS_API_TOKEN_GITHUB }}
         with:
           source_file: 'README.md'
           destination_repo: 'sourcefuse/arc-docs'
-          destination_folder: 'docs/arc-iac-docs/modules/terraform-module-aws-bootstrap'
+          destination_folder: 'docs/arc-iac-docs/modules/terraform-aws-ref-arch-bootstrap'
+          user_email: 'github-actions@github.com'
+          user_name: ${{ github.actor }}
+          commit_message: ${{ github.event.head_commit.message }}
+      - name: Pushes static folder
```

Comment:
Static folder is not there in the repo then there is no need to add this workflow for pushing static folder

Reply:
let me update it, it has image folder

Reply:
Okay